config/jobs/ci-infra/build-ci-infra-images.yaml

```diff
@@ -0,0 +1,49 @@
+postsubmits:
+  gardener/ci-infra:
+  - name: post-build-ci-infra-images
+    cluster: gardener-prow-trusted
+    # skip config/prow folder that build and autobumper do not end up in an endless loop
+    skip_if_only_changed: '^config/prow/'
```

Comment:
`autobump{-jobs}` also updates `hack/{bootstrap,check}-config.sh` and `config/mkpj.sh`. We need to exclude these as well or only trigger if builds if relevant files changed.